src/import/engine.rs

```diff
@@ -256,6 +256,12 @@ impl SSTWriter {
         })
     }
 
+    fn tune_config(env: Arc<Env>, cf_opts: &mut ColumnFamilyOptions) {
+        use DBCompressionType::{Lz4, No};
+        cf_opts.set_env(env);
+        cf_opts.compression_per_level(&[No, No, No, No, No, No, Lz4]);
```

Comment:
this may cause a higher IO which may cause IO becoming the bottleneck, is it?

Reply:
The IO usage will increase but will not be the bottleneck. It's a trade-off.

Reply:
It's better if it is configurable.

Reply:
We are decreasing the unnecessary configuration of TiKV-Importer to improve usability. I don't think we really need to config the compaction algorithm.

Reply:
Addressed.
There is an exists configuration to archive this.